tools/ci_connector_ops/ci_connector_ops/pipelines/pipelines/metadata.py

```diff
@@ -113,13 +89,13 @@ async def run_metadata_lib_test_pipeline(
     async with dagger.Connection(DAGGER_CONFIG) as dagger_client:
         metadata_pipeline_context.dagger_client = dagger_client.pipeline(metadata_pipeline_context.pipeline_name)
         async with metadata_pipeline_context:
-            test_lib_step = TestPoetryModule(
+            test_lib_step = PoetryRun(
```

Comment:
@bnchrch let me know if the creation of a  `PoetryRunPytest(PoetryRun)` step calling `pytest`  would make this code more readable.

Reply:
I think it would, or at least would be a good way to reduce line of code. But not nessesary at all!